src/main/java/com/fasterxml/jackson/core/json/ReaderBasedJsonParser.java

```diff
@@ -2538,8 +2540,8 @@ private final int _skipWSOrEnd() throws IOException
                 _currInputRowStart = _inputPtr;
             } else if (i == INT_CR) {
                 _skipCR();
-            } else if (i != INT_TAB) {
-                _throwInvalidSpace(i);
+            } else if (i != INT_TAB && ((_features & FEAT_MASK_ALLOW_CONTROL_CHAR) != 0 && i != INT_RS)) {
```

Comment:
I think latter `&&` needs to be `||` (so failure if Not Tab and either Don't Allow RS OR not RS).
Otherwise fails some unit tests on CI (see failures)

Reply:
Fixed.